admin_guide/metrics_guidlines.adoc

```diff
@@ -0,0 +1,77 @@
+[[metrics-guidelines]]
+= OpenShift Container Platform (OCP) Metrics Guidlines
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift Container Platform Metrics enable OCP users and administrators to monitor
+pods in OCP installation and quickly get information about specific pod resource utilization
+and based on that decide is it necessary to give additional resources to specific application pods in order to
+avoid capacity and performance issues.
+
+== Metrics Guidelines
+This document gives guidlines for OCP metrics deployment. For specific OCP metrics configuration
+aspets refer to OCP documentation ( Link to OCP v3.4 Administation guide )
+
+=== OpenShift Container Platform Metrics Node Capacity Planning
+
+Running OCP metrics nodes is recommended on separate OCP nodes than application ones.
+For best performance, either ensure OCP metrics pods run on `infra` nodes, or OCP metrics pods ran on
+dedicated OCP nodes
+In our tests, metrics pods were collocated with other  `infra` pods and OCP node capacity was 8 CPUs / 32 GB of memory
+Above recommendations is based on experiences performend during testing
+
+=== Storage recommendations for OpenShift Metrics Deployment
+
+It is necessary prior to metrics deployment to plan storage space for Cassandra
+pod within OpenShift Container Platform metrics. All metrics data are written
+to Cassandra database and it can grow up over time and the grow rate and size of
+Cassandra database is directly proportional to number of pods and nodes in OCP
+cluster monitored by OCP Metrics
+
+It is strongly recommended to run OCP metrics with persistent storage enabled with
+```
+USE_PERSISTANT_STORAGE=true
+```
+Running OCP metrics with `USE_PERSISTANT_STORAGE=true` will ensure data persistance
+across metrics pods lifetime.
+
+In tests performed with with 210 and 990 OCP nodes where 10500 pods and 11000 pods were
+ monitored respectively,it was noticed that cassandra database will grow aproximatelly
+ at speed rate as showed in Table 1. below
+
+.Cassandra Database storage requirements depending on number of nodes / pods in cluster
+[options="header,footer"]
+|====================================================================================
+|Number of Nodes| Number of Pods | Cassandra Storage grow speed |  Cassandra storage grow / day | Cassandra storage size / week
+| 210           | 10500          | 500MB / hour   |   15 GB                       |   75 GB
+| 990           | 11000          | 1 GB / hour    |   30 GB                       |   210 GB
+|====================================================================================
+
+In above caclulation approximatelly 10% of expected size was addedd as overhead.
```

Comment:
to be on safe side with storage requirement, on exact storage size requirement, I am adding as buffer ( just to be sure ) add additional space 10-20% of required storage space. I can however, stick to exact value, but I am opinion adding 10-20% additional storage to storage size should not be issue. Opinions?